calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -19,6 +20,15 @@ public MainPageViewModel()
 
         #region Methods
         // TODO
+        public void ShowNumberOnDisplay(string textToDisplay)
+        {
+            if (DisplayContent.StartsWith("0"))
+            {
+                DisplayContent = "";
```

Comment:
Why do you want it to display as empty instead of `0`?

Reply:
Ah, it's so you don't have leading `0`'s in the display. Perhaps we could do something with converting the DisplayContent to a `decimal` data type and converting it back to a `string`. That would remove any leading `0`'s as well!